app/src/pages/inside/filtersPage/filterTable/filterTableItem/filterTableItem.jsx

```diff
@@ -94,9 +94,9 @@ export const FilterTableItem = ({
       <div className={cx('separator')} />
     </div>
     <div className={cx('block', 'delete-block')}>
-      <div className={cx('bin-icon', { disabled: !canBeDeleted })} onClick={onDelete}>
-        {Parser(BinIcon)}
-      </div>
+      <button className={cx('delete-button')} disabled={!canBeDeleted} onClick={onDelete}>
```

Comment:
What the purpose to wrap div with additional button tag?

Reply:
Now interface component got an outline in focus

Reply:
Button is necessary for control disabled status. So <div> just rendered like disabled, but still clickable. Fixed button outline.

Reply:
You can prevent click on a div with disabled status just controlling onClick callback execution.
 something like onClick={!disabled && onDelete}

Reply:
Thank you for suggestion. Fixed